routingpy/routers/graphhopper.py

```diff
@@ -510,7 +510,7 @@ def isochrones(
             type,
             intervals[0],
             buckets,
-            center,
+            locations,
```

Comment:
I think there's a couple of things here: the parameter is `locations` when really, it's expecting one location as `[lon, lat]` so it already represents a single location. On top of that, `center` is really just that location but the coordinates are represented as strings:

https://github.com/gis-ops/routingpy/blob/73bf55cdcb8871c1c6b9886bc14a5ca8aef9a38f/routingpy/routers/graphhopper.py#L490

The real problem seems to be the confusion about the name of the `locations` parameter: Valhalla, ORS, GraphHopper and HERE all only accept one single location, yet that parameter is consistently named in the plural form. We could either consistently change them to `location` or we accept multiple and fire multiple requests.

Reply:
What adds even more to the confusion is that Valhalla takes one location but nested into an array, but GraphHopper and ORS take one location (unnested).

Reply:
> Valhalla, ORS, GraphHopper and HERE all only accept one single location

Almost: Valhalla & ORS do accept multiple locations, but only ORS is doing what the code assumes. Since we just came from there when we started routingpy, we tried to support that. In the end, I agree, let's just get rid of supporting multiple locations and accept only a single location for isochrones. Major version sort of thing, but no problem to push one, I don't really care tbh.